packages/client/mutations/RemoveOrgUsersMutation.ts

```diff
@@ -220,6 +222,13 @@ export const removeOrgUsersNotificationOnNext: OnNextHandler<
       history.push('/meetings')
       return
     }
+  } else {
+    const removedUserCount = removedUserIds.length
+    atmosphere.eventEmitter.emit('addSnackbar', {
+      key: `removedFromOrg:${affectedOrganizationId}`,
+      autoDismiss: 10,
+      message: `${removedUserCount} ${plural(removedUserCount, 'member')} have been removed from ${affectedOrganizationName}`
```

Comment:
-1 if we're going through the trouble of using `plural` here, we should change `have` to `has` as well.